app/src/main/res/layout-sw600dp/story_chapter_view.xml

```diff
@@ -36,15 +36,15 @@
         android:layout_height="match_parent"
         android:layout_gravity="center_horizontal"
         android:visibility="@{viewModel.index == (viewModel.totalChapters - 1) ? View.INVISIBLE : View.VISIBLE}"
-        app:color="@{viewModel.chapterSummary.chapterPlayState != ChapterPlayState.COMPLETED ? @color/color_def_grey : @color/oppia_primary}" />
+        app:color="@{viewModel.chapterSummary.chapterPlayState != ChapterPlayState.COMPLETED ? @color/component_color_story_activity_story_chapter_card_not_start_stroke_color : @color/component_color_story_activity_story_chapter_card_start_stroke_color}" />
 
       <ImageView
         android:id="@+id/progress_image_view"
         android:layout_width="32dp"
         android:layout_height="32dp"
         android:layout_gravity="center_horizontal"
         android:importantForAccessibility="no"
-        playState:image="@{viewModel.chapterSummary.chapterPlayState}" />
+        playState:image="@{viewModel.chapterSummary.chapterPlayState}"/>
```

Comment:
Space Before `/>`

Press `Control + Alt + L` - reformatting shortcut to fix the formatting.

Reply:
Done.